### DISCUSSION TITLE
Como Gerar o grupo de informações de cobrança da nota fiscal?

### DISCUSSION THREAD

**Discussion by asjsandro** [TadaSoftware/PyNFe#349] [General]
![image](https://github.com/user-attachments/assets/99ec2367-0502-4a24-bf02-4c21e910bb58)

**Comment by leogregianin** [comment]
É necessário implementar o grupo de informações de cobrança com faturas e duplicatas. Vou mover essa discussão para uma issue.

**Comment by leogregianin** [comment]
Como referência a issue aberta: https://github.com/TadaSoftware/PyNFe/issues/350
